src/app/screens/Register/i18n.ts

```diff
@@ -0,0 +1,31 @@
+import i18next from 'i18next';
+
+i18next.addResources('es', 'Register', {
+  logoAlt: 'logo Wolox',
+  firstName: 'Nombre',
+  lastName: 'Apellido',
+  email: 'Correo',
+  password: 'Contraseña',
+  passwordConfirmation: 'Confirmación de Contraseña',
+  btnLogin: 'Iniciar Sesión',
+  btnRegister: 'Registrarme',
+  btnLanguage: 'Ingles',
+  errorEmail: 'Asegúrece de escribir un formato de correo valido',
+  errorRequired: 'Asegúrece de diligenciar este campo',
```

Comment:
same, this could be general `required` with a simplier messaage like `Este campo es requerido`